YouVsVirus/Assets/Scenes/Levelgethome/CreatePopLevelgethome.cs

```diff
@@ -44,7 +46,7 @@ public class CreatePopLevelgethome : MonoBehaviour
         /// <summary>
         /// Number of instantiated NPCs.
         /// </summary>
-        private int npcNumber = 42;
+        public int npcNumber = 42;
```

Comment:
Why does this have to be public?

Reply:
This way, we can easily control the number of NPCs from the unity scene view.
Which makes the whole script reusable for other levels.
Otherwise, we either have to copy the script, just to change the number. Or have to do
some "if name == 'MyLevel' " stuff which is error prone.